package.json

```diff
@@ -278,6 +278,7 @@
     "react-intl": "^2.8.0",
     "react-is": "^16.8.0",
     "react-moment-proptypes": "^1.7.0",
+    "react-motion": "^0.5.2",
```

Comment:
I don't think we should bring this to Kibana. The latest version was released 3 years ago. The library seems to be abandoned: https://github.com/chenglou/react-motion/issues/569